app/views/leads/show.html.erb

```diff
@@ -0,0 +1,69 @@
+<p id="notice"><%= notice %></p>
```

Comment:
All data models use the same show pattern, see Otu's show for how that is rendered out.  You'll need to move these fields to a partial in part.